### ISSUE TITLE
Type 'typeof CBC' is missing the following properties from type 'BlockCipherMode': mixIn, clone

### ISSUE THREAD

**Issue by cos60** [entronad/crypto-es#9]
This is the version information, as follows。
"crypto-es": "^1.2.6",
"typescript": "^4.0.3",

Is that the issue from?
![image](https://user-images.githubusercontent.com/25264968/97416401-58e67900-1941-11eb-8634-849772d3ce98.png)

**Comment by PavolSvidran** [comment]
Why is this closed, when issue is stil there yea it work with CryptoES.mode.CBC as any but still..

**Comment by entronad** [comment]
@PavolSvidran 

There seems no such issue ?
```
const t9: CryptoES.mode.CBC = new CryptoES.mode.CBC();
t9.mixIn;
t9.clone;
```

**Comment by entronad** [comment]
This is caused by our type error in tds that mode should be a class, not an object.
We have fixed it in v1.2.7, Please upgrade and have a retry.
